examples/test.html

```diff
@@ -58,13 +58,18 @@
   attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
   }).addTo(map);
 
+  var layers = [{
+      sql: 'select * from ne_10m_populated_places_simple_3',
+      cartocss: document.getElementById('choropleth').innerHTML,
+  }];
+
   var cartoDBd3Layer = new L.CartoDBd3Layer({
     user: 'fdansv',
-    table: 'ne_adm0_europe',
-    cartocss: document.getElementById('choropleth').innerHTML,
-    tiler_template: "http://development.localhost.lan:8181"
+    layers: layers,
+    styles: layers.map(function(l) { return l.cartocss }),
```

Comment:
wouldn't it make sense for the provider to take the style from the layer definition?